pkg/apis/redhatcop/v1alpha1/groupsync_types.go

```diff
@@ -272,14 +272,6 @@ type SecretRef struct {
 	Key string `json:"key,omitempty"`
 }
 
```

Comment:
what happened here, you are not setting the state anymore?

Reply:
Fixed